src/frontend/komponenter/Fagsak/Søknad/LeggTilBarn.tsx

```diff
@@ -39,33 +38,56 @@ const LeggTilBarn: React.FunctionComponent<IProps> = ({ settSøknadOgValider, s
             process.env.NODE_ENV === 'development'
         ) {
             settPerson({ status: RessursStatus.HENTER });
-            axiosRequest<IPerson, void>({
-                method: 'GET',
-                url: '/familie-ba-sak/api/person/enkel',
-                headers: {
-                    personIdent: ident.verdi,
-                },
-            }).then((hentetPerson: Ressurs<IPerson>) => {
-                settPerson(hentetPerson);
-                if (hentetPerson.status === RessursStatus.SUKSESS) {
-                    const barn: IBarnMedOpplysninger = {
-                        ident: hentetPerson.data.personIdent,
-                        navn: hentetPerson.data.navn,
-                        fødselsdato: hentetPerson.data.fødselsdato,
-                        inkludertISøknaden: true,
-                        manueltRegistrert: true,
-                    };
-                    søknad.barnaMedOpplysninger.push(barn);
-                    settSøknadOgValider(søknad);
 
-                    settVisModal(false);
-                } else if (hentetPerson.status === RessursStatus.FEILET) {
-                    settFeilmelding(hentetPerson.frontendFeilmelding);
+            axiosRequest<IRestTilgang, { brukerIdent: string }>({
+                method: 'POST',
+                url: '/familie-ba-sak/api/tilgang',
+                data: { brukerIdent: ident.verdi },
+            }).then((ressurs: Ressurs<IRestTilgang>) => {
```

Comment:
Dette kunne jeg flyttet ut, men gjorde det ikke da komponenten ikke var så stor. Vi har egentlig ikke diskutert hva vi skal gjøre med slik logikk, kanskje det bør diskuteres? 

Om jeg hadde flyttet den ut måtte jeg uansett sendt med noen callbacks eller flyttet all state til en custom hook. Det er kanskje mest riktig? Isåfall bør den ligge på samme nivå i mappestrukturen eller i en egen mappe?